tiny-firmware/firmware/fsm.c

```diff
@@ -633,4 +633,34 @@ void fsm_msgTxAck(TxAck *msg) {
     }
     layoutHome();
     return;
-}
\ No newline at end of file
+}
+
+void fsm_msgBitcoinTxAck(BitcoinTxAck *msg) {
```

Comment:
This PR adds an address functionality, this should be removed and added, as separate PR

Reply:
Also, don't forget to put this function to in a new file bitcoin_fsm.c

Reply:
It should be declared, because without it I got the following error:
```skycoinProject/hardware-wallet/tiny-firmware//protob/c/messages_map.h:34:110: error: 'fsm_msgBitcoinTxAck' undeclared here (not in a function); did you mean '_BitcoinTxAck'?```